### DISCUSSION TITLE
Which sensor to show output flow from battery in W

### DISCUSSION THREAD

**Discussion by ukshark** [wills106/homeassistant-solax-modbus#167] [Q&A]
I can see the power (live not total) from PV1/2 and combined, i can see power to and from grid in W, but cant find the sensor name for power coming from the battery (the one you see on the App), can anyone help, i want to make it a dial on my dashboard.
I have X1 Hyb gen4

**Comment by ukshark** [comment]
Is it "battery power charge" ?

**Reply by wills106** [reply]
Yes.

Although if it's for the Energy Dashboard you will want to use SolaX Battery Input Energy Today / SolaX Battery Output Energy Today
